platform/upgrades/adding-custom-cloud-credentials.md

```diff
@@ -80,21 +80,21 @@ The final step is to add all the Grid policies to your account. That means that
 4. Click on "Add permissions"
 5. Click on "Attach existing policies directly"
 
-![Granting permissions to an user.](../../.gitbook/assets/image%20%2813%29.png)
+![Granting permissions to a user.](../../.gitbook/assets/image%20%2813%29.png)
 
 1. Search for the policy IAMFullAccess:  
 2. Click the Check Box to the left of `IAMFullAccess`
-3. Click on "Next:Review"
+3. Click on "Next: Review"
 4. Click on "Add permissions"
 
 Now that you have added the right permissions to your user name, you can use the user's AWS API keys with Grid.
 
 ### Step 3: Create Role & Policy grid requires
 
-For the next step you're going to create role we're going to assume into. For this you'll be using terraform. Make sure you have `git`, `terraform`, `jq` and `AWS CLI` installed on your machine. Installation instruction of these tools are [available](adding-custom-cloud-credentials.md#installing-3rd-party-tools). If you're familiar with terraform we recommend you check the terraform module we'll be using to create necessary roles & policies. [https://github.com/gridai/terraform-aws-gridbyoc](https://github.com/gridai/terraform-aws-gridbyoc) This module is published on official terraform registry for your convenience [https://registry.terraform.io/modules/gridai/gridbyoc/aws/latest](https://registry.terraform.io/modules/gridai/gridbyoc/aws/latest)
+For the next step, you're going to create a role using terraform. Make sure you have `git`, `terraform`, `jq`, and `AWS CLI` installed on your machine. Installation instruction of these tools are [available](adding-custom-cloud-credentials.md#installing-3rd-party-tools). If you're familiar with terraform we recommend you check the terraform module we'll be using to create necessary roles & policies. [https://github.com/gridai/terraform-aws-gridbyoc](https://github.com/gridai/terraform-aws-gridbyoc) This module is published on the official terraform registry for your convenience [https://registry.terraform.io/modules/gridai/gridbyoc/aws/latest](https://registry.terraform.io/modules/gridai/gridbyoc/aws/latest)
```

Comment:
Expand this a bit what's the significance of this role

Reply:
Should we mention AWS ARN role?

Reply:
ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to.

Reply:
ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to.

Reply:
ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to.

Reply:
ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to. Not sure what you're referring to by mentioning it.